storybook/src/data/lists.data.js

```diff
@@ -1,3344 +1,109 @@
-        user: 'dcaress97@seesaa.net',
-        date: '1/8/2019',
-      },
-      {
-        company_name: 'Omba',
-        department: 'Training',
-        city: 'Zuru',
-        user: 'kfreeberne98@tripadvisor.com',
-        date: '12/23/2018',
-      },
-      {
-        company_name: 'Lajo',
-        department: 'Marketing',
-        city: 'Xingguo',
-        user: 'eitzhaki99@wired.com',
-        date: '6/21/2019',
-      },
-      {
-        company_name: 'Dabvine',
-        department: 'Marketing',
-        city: 'Parumasan',
-        user: 'spoynzer9a@rambler.ru',
-        date: '10/24/2018',
-      },
-      { company_name: 'Wikido', department: 'Training', city: 'Pende', user: 'amclane9b@noaa.gov', date: '1/3/2019' },
-      {
-        company_name: 'Layo',
-        department: 'Product Management',
-        city: 'Jixian',
-        user: 'lvarnam9c@liveinternet.ru',
-        date: '4/25/2019',
-      },
-      {
-        company_name: 'Flipbug',
-        department: 'Engineering',
-        city: 'Lincha',
-        user: 'tsifleet9d@ask.com',
-        date: '9/2/2018',
-      },
-      {
-        company_name: 'Voonte',
-        department: 'Product Management',
-        city: 'Daleman',
-        user: 'jking9e@hatena.ne.jp',
-        date: '11/21/2018',
-      },
-      {
-        company_name: 'Babblestorm',
-        department: 'Human Resources',
-        city: 'Lingdi',
-        user: 'mhabble9f@w3.org',
-        date: '6/11/2019',
-      },
-      {
-        company_name: 'Podcat',
-        department: 'Services',
-        city: 'Çobansığnaq',
-        user: 'bfattore9g@a8.net',
-        date: '5/27/2019',
-      },
-      {
-        company_name: 'Aimbo',
-        department: 'Support',
-        city: 'Spassk',
-        user: 'rbrigden9h@booking.com',
-        date: '8/1/2018',
-      },
-      {
-        company_name: 'Youfeed',
-        department: 'Support',
-        city: 'Xianyan',
-        user: 'efarfoot9i@ycombinator.com',
-        date: '6/7/2019',
-      },
-      {
-        company_name: 'Edgeify',
-        department: 'Business Development',
-        city: 'Leyuan',
-        user: 'dblunden9j@ibm.com',
-        date: '1/18/2019',
-      },
-      {
-        company_name: 'Jazzy',
-        department: 'Accounting',
-        city: 'Aigínio',
-        user: 'fstavers9k@flavors.me',
-        date: '7/11/2018',
-      },
-      {
-        company_name: 'Flashpoint',
-        department: 'Research and Development',
-        city: 'Dewetsdorp',
-        user: 'bmacphail9l@parallels.com',
-        date: '6/10/2019',
-      },
-      {
-        company_name: 'Omba',
-        department: 'Research and Development',
-        city: 'Huangzhou',
-        user: 'kmaddigan9m@phpbb.com',
-        date: '6/5/2019',
-      },
-      {
-        company_name: 'Edgewire',
-        department: 'Training',
-        city: 'Chattanooga',
-        user: 'avedishchev9n@marketwatch.com',
-        date: '10/22/2018',
-      },
-      {
-    
```

Comment:
Since you're declaring a ver here you might as well use key names to use shorthand object initialization afterwards ie `.setProperties({company_name, department, ... })`

Reply:
all these lines with similar logics immediately make me raise 🎏 
you should introduce something like: `const random = (arr) => arr[Math.floor(Math.random() * arr.length)];`
then your code could be as simple as
```
 .setProperties({
      company_name: random(companyNames),
      department: random(departments),
      ...
    })
```

Reply:
It is a much cleaner approach, indeed.